### PR TITLE
Update Send & Execute XCM docs for XCM V3

### PR DESCRIPTION
**Pull request by eshaben**
### Description

This PR updates the Send & Execute XCM docs for XCM V3 and Weights V2. It also uses Polkadot.js API snippets instead of using the Polkadot.js Apps interface.

There were new snippets added, but keeping the old ones for now for the CN version. Images are no longer needed for this page, but again, keeping any images for CN version until translations are complete.

⚠️⚠️ Will circle back to the checklist after an initial review ⚠️⚠️ 

### Checklist

- [ ] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects
- [ ] If pages have been moved around, I have run the `move-pages.py` script to move the pages and update the image paths on the chinese repo
    - [ ] After the script has been run, I have created an additional PR in `moonbeam-docs-cn`
- [ ] If images have been added, I have run the `compress-images.py` script to compress the images.
- [ ] If variables (in variables.yml) need to be updated (such as a name change), I have updated the `moonbeam-docs-cn` repo to use the new variables
- [ ] If this page requires a disclaimer, I have added one

### Corresponding PRs

Please link to any corresponding PRs here.

### After Translation Requirements

- [ ] Will need to create PR in `moonbeam-docs` repo to remove images
- [ ] Will need to create PR in `moonbeam-docs` repo to remove variables
- [ ] Will need to create PR in `moonbeam-mkdocs` repo to add redirects for Chinese site
- [ ] No additional PRs are required after the translations are done

#### Items to be Updated

Please list any of the items that will need to be added or deleted after the translations are done here.

### CODE REVIEW

File: .snippets/code/polkadotXcm/xcmSend/sendWithPolkadot.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+};
+const instr3 = {
+  DepositAsset: {
+    assets: { Wild: 'All' },
+    beneficiary: {
+      parents: 1,
+      interior: {
+        X1: {
+          AccountId32: {
+            id: RELAY_ACC_ADDRESS,
```
[the remark this answers is not in the document]
Comment (eshaben) [reply .snippets/code/polkadotXcm/xcmSend/sendWithPolkadot.js:31]: It is 32-bytes... I used the tool to decode it. What error are you getting?

[the remark this answers is not in the document]
Comment (albertov19) [reply .snippets/code/polkadotXcm/xcmSend/sendWithPolkadot.js]: Nah just saying that I used the encoded version `5Hz....` and it failed on me. So we need to explicitly say that you need to use the decoded version.

[the remark this answers is not in the document]
Comment (eshaben) [reply .snippets/code/polkadotXcm/xcmSend/sendWithPolkadot.js]: oh I thought you were saying you used the encoded call data, which uses an already decoded address, and you got that error 😂 

ok ok this makes sense. well anyways, I made the update to add the decodeAddress(address) part in my commit yesterday

File: .snippets/code/polkadotXcm/xcmSend/encodedCalldata.js [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
 const providerWsURL = 'wss://wss.api.moonbase.moonbeam.network';
 // Set parents to 1 and interior to here to target the relay chain
-const relayChainMultilocation = { parents: 1, interior: null } 
+const relayChainMultilocation = { parents: 1, interior: null };
 const amount = BigInt(1 * 10 ** 12); // 1 UNIT
-const bob = '0x0c36e9ba26fa63c60ec728fe75fe57b86a450d94e7fee7f9f9eddd0d3f400d67';
+// You can use the decodeAddress function to ensure that your address is properly
+// decoded. If it isn't decoded, it will decode it and if it is, it will ignore it
+const bob = decodeAddress(
+  '0x0c36e9ba26fa63c60ec728fe75fe57b86a450d94e7fee7f9f9eddd0d3f400d67'
```
Comment (albertov19) [inline .snippets/code/polkadotXcm/xcmSend/encodedCalldata.js:12]: Here is better if you provde the encoded address right? As that is the one that readers/devs are used to see?

**Review by albertov19 — APPROVED** [review]
LGTM! Last minor nitpick but I've approved

**Pull request merged**